cdk-testsuite/config/config.yaml

```diff
@@ -0,0 +1,26 @@
+path_linux : '/home/amit/openshift/rc5/cdk/components/rhel/misc/shared_folder/rhel-ose'
+path_win : ''
+path_mac : ''
+openshift_lib_MODULE : '/home/amit/workspace/cdk-testsuite/libraries/openshift/openshift.py'
```

Comment:
@ashetty1 These path will be changed as per the cdk vagrant file directory and clone directory where you cloned the repo..
Example : 
supposed i download cdk in /home/cdktest then linux_path would be /home/cdktest/${VAGRANT_PATH}'. So this path will be changed as per the environment you are running the testcases. Same applies for  "openshift_lib_MODULE"